codegen/import.go

```diff
@@ -29,3 +33,82 @@ func (s *ImportSpec) Code() string {
 	}
 	return fmt.Sprintf(`"%s"`, s.Path)
 }
+
+// GetMetaTypeInfo gets type and import info from an attributes metadata. struct:field:type can have 3 arguments,
```

Comment:
Should `GetMetaTypeInfo` be exported? I don't see it being used outside the `codegen` package. If it is not used anywhere else please unexport the function.